### ISSUE TITLE
remove / upgrade fragile use of `lower.age.limit`

### ISSUE THREAD

**Issue by aarathybabu97** [njtierney/conmat#77]
- predict_contacts.R

**Comment by aarathybabu97** [comment]
```r
  # this could be changed to a function for lower age limit
  age_min_integration <- min(population$lower.age.limit)
  bin_widths <- diff(population$lower.age.limit)
  final_bin_width <- bin_widths[length(bin_widths)]
  age_max_integration <- max(population$lower.age.limit) + final_bin_width
```

**Comment by njtierney** [comment]
Could potentially provide a helper cleaner function that takes in a dataset and changes the names from whatever is identified as the lower.age.limit to `lower.age.limit`. This could then be inserted before key parts of code, and means user has control of it, but we don't need to change downstream code too much?

e.g.,

```r
prepare_age_names(data, age_col = "my_age", population_col = "pop")
```

Would return a dataset with the `age_col` and `population_col` being named "lower.age.limit" and "population", respectively. 

Not the best solution, but could be the least-worst

**Comment by njtierney** [comment]
fixed in #93
